src/integrations/prefect-dask/README.md

```diff
@@ -0,0 +1,260 @@
+14:10:09.857 | INFO    | distributed.scheduler - Receive client connection: Client-8c1e0f24-9133-11ed-800e-86f2469c4e7a
+14:10:09.859 | INFO    | distributed.core - Starting established connection to tcp://127.0.0.1:59516
+14:10:09.862 | INFO    | prefect.task_runner.dask - The Dask dashboard is available at http://127.0.0.1:8787/status
+14:10:11.344 | INFO    | Flow run 'chocolate-pony' - Created task run 'read_data-5bc97744-0' for task 'read_data'
+14:10:11.626 | INFO    | Flow run 'chocolate-pony' - Submitted task run 'read_data-5bc97744-0' for execution.
+14:10:11.795 | INFO    | Flow run 'chocolate-pony' - Created task run 'process_data-090555ba-0' for task 'process_data'
+14:10:11.798 | INFO    | Flow run 'chocolate-pony' - Submitted task run 'process_data-090555ba-0' for execution.
+14:10:13.279 | INFO    | Task run 'read_data-5bc97744-0' - Finished in state Completed()
+14:11:43.539 | INFO    | Task run 'process_data-090555ba-0' - Finished in state Completed()
+14:11:43.883 | INFO    | Flow run 'chocolate-pony' - Finished in state Completed('All states completed.')
+```
+
+## Resources
+
+For additional examples, check out the [Usage Guide](usage_guide)!
+
+### Installation
+
+Get started by installing `prefect-dask`!
+
+=== "pip"
+
+    ```bash
+    pip install -U prefect-dask
+    ```
+
+=== "conda"
+
+    ```bash
+    conda install -c conda-forge prefect-dask
+    ```
+
+Requires an installation of Python 3.7+.
+
+We recommend using a Python virtual environment manager such as pipenv, conda, or virtualenv.
+
+These tasks are designed to work with Prefect 2. For more information about how to use Prefect, please refer to the [Prefect documentation](https://docs.prefect.io/).
+
+### Feedback
+
+If you encounter any bugs while using `prefect-dask`, feel free to open an issue in the [prefect](https://github.com/PrefectHQ/prefect) repository.
+
+If you have any questions or issues while using `prefect-dask`, you can find help in either the [Prefect Discourse forum](https://discourse.prefect.io/) or the [Prefect Slack community](https://prefect.io/slack).
+
+### Contributing
+
+If you'd like to help contribute to fix an issue or add a feature to `prefect-dask`, please [propose changes through a pull request from a fork of the repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+
+Here are the steps:
+
+1. [Fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository)
+2. [Clone the forked repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository)
+3. Install the repository and its dependencies:
+```
+pip install -e ".[dev]"
+```
+4. Make desired changes
+5. Add tests
+6. Insert an entry to [CHANGELOG.md](https://github.com/PrefectHQ/prefect-dask/blob/main/CHANGELOG.md)
```

Comment:
hmm not sure if we should remove the whole section or just the changelog step but rn it refers to the old repo

Reply:
I removed just this step for now.